### PR TITLE
Adding HINTS.md to Zipper exercise

### PR DESCRIPTION
**Pull request by GalaDe**

### CODE REVIEW

File: exercises/zipper/HINTS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Hints
+This exercise requires to deal with custom equality. For more information see
```
Comment (ErikSchierboom) [inline exercises/zipper/HINTS.md]: I should rename "This exercise requires to deal" to "This exercise deals"

**Comment by ErikSchierboom** [comment]
This PR contains two files. You can see where this went wrong by looking at the commit history of this branch: https://github.com/GalaDe/xcsharp/commits/zipper There you see that the zipper change is committed after the sum-of-multiples change, which commit can also be found in the sum-of-multiples branch: https://github.com/GalaDe/xcsharp/commits/sum-of-multiples This means that you branched the zipper branch off the sum-of-multiples branch, instead of branching from the master branch.

**Comment by GalaDe** [comment]
Hi @ErikSchierboom, Yes, looks like I created a zipper branch off the sum-of-multiple branch. Another lesson was learned. Sorry again for inconvenience.

**Comment by ErikSchierboom** [comment]
@GalaDe No problem. Lots of git lessons when working with PR's :)

**Comment by ErikSchierboom** [comment]
@GalaDe Well, yeah, sort of. However, what you also can do is this (assuming you are on this PR's branch):

```
git fetch exercism
git rebase exercism/master
git push --force
```

This will update your branch to the latest master. If that somehow fails, I would do as you suggested.

**Comment by GalaDe** [comment]
Done. Thank you again!

**Pull request closed without merging**